<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UTMDialog</class>
 <widget class="QDialog" name="UTMDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert positions dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Position</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>Convert</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QPushButton" name="LLtoUTM_pushButton">
           <property name="text">
            <string>==&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="UTMtoLL_pushButton">
           <property name="text">
            <string>&lt;==</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>72</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>UTM</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="4" column="1">
          <widget class="QLineEdit" name="UTM_Northing_lineEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Northing [m]:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Zone:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="UTM_Zone_lineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Easting  [m]:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="UTM_Easting_lineEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Number of digits:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="NumOfDigitsUTM_spinBox">
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Geodetic</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>Latitude:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Latitude_lineEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Longitude:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="Longitude_lineEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="NumOfDigitsLL_spinBox">
           <property name="maximum">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of digits:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Decimal-separator</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="Dot_radioButton">
        <property name="text">
         <string>. (dot)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="Comma_radioButton">
        <property name="text">
         <string>, (comma)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Reference ellipsoid</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QComboBox" name="ReferenceEllipsoid_comboBox">
        <item>
         <property name="text">
          <string>Airy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Airy, modified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Australian National</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bessel 1841</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bessel 1841 (Nambia)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clarke 1866</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clarke 1880</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everest, modified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fischer 1960 (Mercury)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fischer 1960, modified</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fischer 1968</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GRS-67</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>GRS-80</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Helmert 1906</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hough</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>International</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Krassovsky</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>South American 1969</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WGS-60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WGS-66</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WGS-72</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>WGS-84</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ETRS-89</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>374</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Map options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Map type</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QRadioButton" name="MapType_1_radioButton">
           <property name="text">
            <string>Satellite map view in daylight mode</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="MapType_3_radioButton">
           <property name="text">
            <string>Satellite map view with streets in daylight mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Zoom level</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QSpinBox" name="ZoomLevel_spinBox"/>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>288</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ClearAll_pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ShowMap_pushButton">
       <property name="text">
        <string>Show map</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>Latitude_lineEdit</tabstop>
  <tabstop>Longitude_lineEdit</tabstop>
  <tabstop>NumOfDigitsLL_spinBox</tabstop>
  <tabstop>LLtoUTM_pushButton</tabstop>
  <tabstop>UTMtoLL_pushButton</tabstop>
  <tabstop>UTM_Northing_lineEdit</tabstop>
  <tabstop>UTM_Easting_lineEdit</tabstop>
  <tabstop>UTM_Zone_lineEdit</tabstop>
  <tabstop>NumOfDigitsUTM_spinBox</tabstop>
  <tabstop>Dot_radioButton</tabstop>
  <tabstop>Comma_radioButton</tabstop>
  <tabstop>MapType_1_radioButton</tabstop>
  <tabstop>MapType_3_radioButton</tabstop>
  <tabstop>ZoomLevel_spinBox</tabstop>
  <tabstop>ClearAll_pushButton</tabstop>
  <tabstop>ShowMap_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
